packages/assets-controllers/src/TokensController.ts

```diff
@@ -604,7 +605,7 @@ export class TokensController extends BaseController<
     const suggestedAssetMeta = {
       asset,
       id: this._generateRandomId(),
-      status: SuggestedAssetStatus.pending as SuggestedAssetStatus.pending,
+      status: SuggestedAssetStatus.Pending as SuggestedAssetStatus.Pending,
```

Comment:
It can also be solved by using `as const` apparently, which is a better solution in my opinion. I hadn't looked at this code specifically, it's just part of an automatic refactor.